olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/PartitionsRequest.h

```diff
@@ -146,9 +114,9 @@ class DATASERVICE_READ_API PartitionsRequest final {
    * @param none
```

Comment:
```suggestion
    * @param layer_id
```
Same here - please, add corresponding doxygen comment